v3/src/components/graph/models/graph-content-model.ts

```diff
@@ -270,6 +270,17 @@ export const GraphContentModel = DataDisplayContentModel
         const updateCategoriesOptions = self.getUpdateCategoriesOptions()
         self.adornmentsStore.updateAdornments(updateCategoriesOptions)
       }, {name: "GraphContentModel.afterAttachToDocument.updateAdornments"}, self.dataConfiguration))
+
+      // When showMeasuresForSelection is true, update adornments when selection changes
+      addDisposer(self, mstReaction(() => {
+        return self.dataConfiguration.selection
+      },
+        () => {
+          if (self.dataConfiguration.showMeasuresForSelection) {
```

Comment:
You also presumably want to run the reaction when `showMeasuresForSelection` changes, in which case you could do something like:
```suggestion
        return [self.dataConfiguration.showMeasuresForSelection, self.dataConfiguration.selection]
      },
        ([showMeasuresForSelection, selection]) => {
          if (showMeasuresForSelection) {
```
and then you probably want `equals: comparer.structural` in the options below.